packages/app/src/backend/device-service.ts

```diff
@@ -149,6 +149,11 @@ export class DeviceService {
     method: Method.Delete
     body: GetThreadBody
   }): Promise<RequestResponse>
+  public request(config: {
+    endpoint: Endpoint.Messages
+    method: Method.Delete
+    body: GetTemplateBody
```

Comment:
It's a delete method, so let's use the `DeleteTemplateBody`

Reply:
There was already a proper request. I removed this one [f9d5dde](https://github.com/mudita/mudita-center/pull/770/commits/f9d5ddea9864fd02e95e471b3e5f9c2f4e3b4d8a)